main/utils/autorun/mac-scheduler.js

```diff
@@ -0,0 +1,55 @@
+const { app } = require('electron')
+const { writeFile, unlink } = require('fs').promises
+const { spawnSync } = require('child_process')
+const { join } = require('path')
+const log = require('electron-log')
+
+const { getBinaryPath, getAutorunHomeDir } = require('../paths')
+
+const pathToProbeCLI = getBinaryPath()
+
+function launchctl(command, args) {
+  return spawnSync('launchctl', [command, ...args])
+}
+
+module.exports = {
+  get: function (taskname, format, verbose) {
+    return new Promise((resolve, reject) => {
+      reject()
+    })
+  },
+  create: function (taskname, taskrun) {
+    return new Promise((resolve, reject) => {
+      const plistTemplate = require('./taskTemplateMac')
+      const taskPlistStr = plistTemplate({
+        taskName: taskname,
+        taskCmd: pathToProbeCLI,
+        OONI_HOME_autorun: getAutorunHomeDir()
+      })
+      const taskPlistPath = join(
+        app.getPath('home'), 'Library', 'LaunchAgents',
+        `${taskname}.plist`
+      )
+      writeFile(taskPlistPath, taskPlistStr).then(() => {
+        log.debug(`Autorun task plist file created: ${taskPlistPath}`)
+        log.debug(`Enable task with: launchctl load ${taskPlistPath}`)
+        const cmd = launchctl('load', [taskPlistPath])
```

Comment:
Are you sure this is the right command? In probe-cli we are instead doing:
```
launchctl enable && launchctl bootstrap
```

Reply:
In most places I read, it is recommended to use `load` because it registers the task and starts it right away.
https://medium.com/swlh/how-to-use-launchd-to-run-services-in-macos-b972ed1e352
https://www.maketecheasier.com/use-launchd-run-scripts-on-schedule-macos/
https://apple.stackexchange.com/questions/29056/launchctl-difference-between-load-and-start-unload-and-stop

Reply:
@bassosimone maybe we want to use the same behaviour on probe-cli then.

Reply:
I think we should not. The `load` command is deprecated and may not work. I think it's safer that `probe-desktop` does what `probe-cli` does, which is what `brew services` does.

Reply:
Thanks @bassosimone for showing how `load/unload` is not the recommended way to do this. I will go ahead and change this.